ocaml/idl/datamodel.ml

```diff
@@ -9200,6 +9202,50 @@ let alert =
     ()
 *)
 
+let net_sriov_create = call
+    ~name:"create"
+    ~doc:"Create a net-sriov on specified pif"
+    ~params:[Ref _pif, "pif", "PIF which to enable sriov";
+             Ref _network, "network", "Network to link SRIOV"]
+    ~result:(Ref _net_sriov, "The reference of the created SRIOV object")
+    ~in_product_since:rel_jura
+    ~allowed_roles:_R_POOL_OP
+    ()
+
+let net_sriov_destroy = call
+    ~name:"destroy"
+    ~doc:"Destroy a net-sriov"
+    ~params:[Ref _net_sriov, "self", "SRIOV to destroy"]
+    ~in_product_since:rel_jura
+    ~allowed_roles:_R_POOL_OP
+    ()
+
+(** network sriov **)
+module NET_sriov = struct
+  let obj =
+    create_obj
+      ~name:_net_sriov
+      ~descr:"network-sriov which connects logical pif and physical pif"
+      ~doccomments:[]
+      ~gen_constructor_destructor:false
+      ~gen_events:true
+      ~in_db:true
+      ~lifecycle:[Published, rel_jura, ""]
+      ~messages:[net_sriov_create; net_sriov_destroy]
+      ~messages_default_allowed_roles:_R_POOL_OP
+      ~persist:PersistEverything
+      ~in_oss_since:None
+      ~contents:
+        ([
+          uid _net_sriov;
+          field ~qualifier:StaticRO ~ty:(Ref _pif) ~in_product_since:rel_jura "physical_PIF" "physical pif which the ner_sriov created" ~default_value:(Some (VRef ""));
```

Comment:
Typo in `ner_sriov`

Reply:
Ah, yes, and also have to update `pif` to uppercase.